src/App.jsx

```diff
@@ -1,28 +1,63 @@
 import React from 'react';
 import './App.scss';
 import './styles/general.scss';
+import { getTodos } from './api';
 import { TodoList } from './components/TodoList';
 import { CurrentUser } from './components/CurrentUser';
 
 class App extends React.Component {
   state = {
     todos: [],
     selectedUserId: 0,
+    randomizedTodos: null,
```

Comment:
you don't need to store randomized todos in the state, calculate them in render method in the TodoList component